presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -139,6 +153,12 @@ Array Functions
         SELECT ngrams(ARRAY['foo', 'bar', 'baz', 'foo'], 5); -- [['foo', 'bar', 'baz', 'foo']]
         SELECT ngrams(ARRAY[1, 2, 3, 4], 2); -- [[1, 2], [2, 3], [3, 4]]
 
+.. function:: none_match(array(T), function(T,boolean)) -> boolean
```

Comment:
It doesn't seem that the functions are actually listed in alphabetical order so it probably makes more sense to keep these three functions together.

Reply:
@rongrong : Looks to me the functions are listed in alphabetical order? Maybe some functions are not following the ordering strictly (although I didn't see it with skim 😄 ) and we should fix them?

Reply:
Maybe it's the array_intersect/union/except that's wrong then. Can you add a follow up PR to fix that? Thanks!

Reply:
also add space here.

Reply:
@rongrong 
> Maybe it's the array_intersect/union/except that's wrong then. Can you add a follow up PR to fix that? Thanks!

Yup: https://github.com/prestodb/presto/pull/13736 😃